tests/waku_filter_v2/test_waku_client.nim

```diff
@@ -1952,164 +2142,8 @@ suite "Waku Filter - End to End":
         await wakuFilter.handleMessage(pubsubTopic, msg)
 
         # Then the client receives the message
-        require await pushHandlerFuture.withTimeout(FUTURE_TIMEOUT)
+        check await pushHandlerFuture.withTimeout(FUTURE_TIMEOUT)
         let (pushedMsgPubsubTopic, pushedMsg) = pushHandlerFuture.read()
         check:
           pushedMsgPubsubTopic == pubsubTopic
           pushedMsg == msg
-
-  suite "MessagePushHandler - Msg List":
```

Comment:
Where this suite is being moved?

Reply:
In the beginning I split tests in two to "optimise" the handlers. That is, one set of tests needed a less complex handler, so I created two different handlers, and split the tests between the two. 
After a while, I realised the advantage of having two handlers (a close to zero optimisation), was not worth the simplicity of having just one handler.